package.json

```diff
@@ -17,23 +17,25 @@
     "url": "shelf.io"
   },
   "files": [
-    "environment.js",
     "jest-preset.js",
-    "setup.js",
-    "teardown.js",
-    "wait-for-localhost.js"
+    "lib/**/*"
   ],
   "scripts": {
+    "build": "rm -rf lib/ && yarn build:types && babel src --out-dir lib --ignore '**/*.test.ts' --extensions '.ts'",
+    "build:types": "tsc --emitDeclarationOnly --declaration --isolatedModules false --declarationDir lib",
     "coverage": "jest --coverage",
     "lint": "eslint . --ext .js,.ts,.json --fix",
     "lint:ci": "eslint . --ext .js,.ts,.json",
-    "test": "export ENVIRONMENT=local && jest ."
+    "prepack": "yarn build",
```

Comment:
Is this script used somewhere?

Reply:
I saw almost every open-source lib uses this command, but I also never found where exactly it is used.
<img width="2861" alt="image" src="https://user-images.githubusercontent.com/19535641/183656918-7eaef05f-0e7d-4613-a4ed-67d549101141.png">

Reply:
This script is used when running `npm publish`:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/31747499/183658190-0136a2c9-37c9-4edb-a859-2b5fae7bfd97.png">
https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-operation-order

Reply:
@harazdovskiy